php/libraries/LorisForm.class.inc

```diff
@@ -1940,8 +1940,12 @@ class LorisForm
     {
         foreach ($submitKeys as $key) {
             if (!isset($this->form[$key])) {
-                $tp = $this->groupMapping[$key];
-                $el = $this->form[$tp['groupName']]['elements'][$tp['index']];
+                $tp = isset($this->groupMapping[$key]) ?
+                    $this->groupMapping[$key] :
+                    null;
+                $el = is_null($tp) ?
```

Comment:
`is_null($tp) ? null : $something;` is not convertible to a null coalesce operator.